spec/graphql/mutations/integrations/okta/create_spec.rb

```diff
@@ -0,0 +1,85 @@
+# frozen_string_literal: true
+
+require 'rails_helper'
+
+RSpec.describe Mutations::Integrations::Okta::Create, type: :graphql do
+  let(:membership) { create(:membership) }
+
+  let(:mutation) do
+    <<-GQL
+      mutation($input: CreateOktaIntegrationInput!) {
+        createOktaIntegration(input: $input) {
+          id,
+          name,
+          code,
+          clientId,
+          clientSecret,
+          domain,
+        }
+      }
+    GQL
+  end
+
+  around { |test| lago_premium!(&test) }
```

Comment:
I guess this is not needed since we don't check regular premium license in services